scripts/model/repo.js

```diff
@@ -0,0 +1,50 @@
+(function(module) {
+  var repos = {};
+  commits = 0;
+  totalCommits = [];
+
+  repos.all = [];
+
+  repos.requestRepos = function(callback) {
+
+    $.ajax({
+      url: 'https://api.github.com/users/aliza89p/repos' +
+      '?per_page=10' +
+      '&sort=updated',
+      type: 'GET',
+      headers: {'Authorization': 'token ' + githubToken},
+      success: function(data) {
+        repos.all = data;
+        callback();
+      }
+    });
+  };
+
+  // repos.all.map(function(repo) {
```

Comment:
The AJAX looks good, you probably just need to find a place to put it. I hope the code review in class gave you some ideas.